projects/storefrontlib/src/cms-components/cart/cart-coupon/applied-coupons/applied-coupons.component.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { Component, Input } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { CartService, I18nTestingModule, Voucher } from '@spartacus/core';
+import { ICON_TYPE } from '../../../../cms-components/misc/icon/index';
+import { AppliedCouponsComponent } from './applied-coupons.component';
+
+const coupon1: Voucher = { code: 'coupon1' };
+const coupon2: Voucher = { code: 'coupon2' };
+
+@Component({
+  selector: 'cx-icon',
+  template: '',
+})
+class MockCxIconComponent {
+  @Input() type: ICON_TYPE;
+}
+
```

Comment:
Need follow what we aligned, add a parent in testing to see the design and also decouple with parent

Reply:
see https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/2484d4d7555c559bf973e976d28bfc322d3b432d/projects/storefrontlib/src/cms-components/myaccount/my-coupons/coupon-card/coupon-card.component.spec.ts as reference

Reply:
Done